src/cloud-api-adaptor/test/e2e/libvirt_test.go

```diff
@@ -96,3 +96,27 @@ func TestLibvirtPodsMTLSCommunication(t *testing.T) {
 	assert := LibvirtAssert{}
 	DoTestPodsMTLSCommunication(t, testEnv, assert)
 }
+
+func TestLibvirtKbsKeyReleaseWithDefaultOpa(t *testing.T) {
+	if !isTestWithKbs() {
+		t.Skip("Skipping kbs related test as kbs is not deployed")
+	}
+	if !isTestWithCustomizedOpa() {
+		t.Skip("Skipping TestLibvirtKbsKeyReleaseWithCustomizedOpa as default opa is used")
+	}
```

Comment:
Are the two skip logics the wrong way around for the two test cases?

Reply:
I'm also trying to work out if there is a way, by delaying the `EnableKbsCustomizedPolicy` step that we could potentially run both test cases in the same test run. Is there a way to use the kbs-client to reset the policy, or get the user to set both `TEE_CUSTOMIZED_KBS_POLICY` and `DEFAULT_KBS_POLICY`, so we can optionally test both?

Reply:
I was thinking about it, but looks it's not reasonable to change kbs when do specific testing case. If there is better approach to test both of the case and keep it graceful I'm glad to change it :-).

Reply:
@stevenhorsman OK, I'm fine with this approach, added in the latest commit.

Reply:
Great, I'll test it out now. Not needing to specify the customise policy is a nice side effect too!